Jellyfin.Api/WebSocketListeners/ActivityLogWebSocketListener.cs

```diff
@@ -56,6 +58,16 @@ protected override void Dispose(bool dispose)
         base.Dispose(dispose);
     }
 
+    private new void Start(WebSocketMessageInfo message)
```

Comment:
Why new? Shouldn't it just be protected virtual?

Reply:
`protected virtual` throws `CS0108`, only way it did not throw this error was using `new`

Reply:
Parent uses protected virtual. Child classes use protected override.